src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -2821,9 +2821,6 @@ private sealed class ComNativeDescriptorProxy : TypeDescriptionProvider
         {
             private readonly TypeDescriptionProvider _comNativeDescriptor;
 
-            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2072:UnrecognizedReflectionPattern",
```

Comment:
I'm not sure how this can be removed when we trim the assembly during the dotnet/runtime build. Is the idea to put it into a LibraryBuild.xml file?

Reply:
The `Type.GetType` would throw at runtime if the type doesn't exist. And that's exactly what linker will assume during library run (dotnet/runtime build) - so the code using the returned type after will not produce any warnings.

This 100% correct for trimming entire apps. For library run it's questionable - it can still be a valid behavior or like in this case it's not exactly what we want. But that's how the linker behaves currently - not sure it's worth implementing some new logic for the library run right now (since it's only used by the runtime repo currently).

Reply:
No plan to do anything about this.

Basically no behavior is "correct":
* If we warn, the linker is basically saying "Please review the type name, I can't verify it for you, but if it's correct then this is noise"
* If we don't warn, there might be a typo in the type name and no tooling would tell you.

So if the type name is correct - then this should definitely not warn. If the type name is not correct - ideally if would warn, but it has no way to know that it's incorrect.